binary/expression.proto

```diff
@@ -101,13 +100,15 @@ message Expression {
     }
 
     message ScalarFunction {
-        Extensions.FunctionId id = 1;
+        // points to a function_surrogate_key defined in this plan
+        uint32 function_reference = 1;
         repeated Expression args = 2;
         Type output_type = 3;
     }
 
     message AggregateFunction {
-        Extensions.FunctionId id = 1;
+        // points to a function_surrogate_key defined in this plan
```

Comment:
Is the `surrogate_key` language still accurate? It looks like  this is referring to things called anchor now.

Reply:
Will fix. I kept it some places since the anchor is a surrogate key but it shouldn't be stated here.

Reply:
fixed